app/src/main/kotlin/net/rafaeltoledo/social/ui/feature/main/MainViewModel.kt

```diff
@@ -0,0 +1,8 @@
+package net.rafaeltoledo.social.ui.feature.main
+
+import net.rafaeltoledo.social.ui.BaseViewModel
+
+class MainViewModel(private val string: String) : BaseViewModel() {
```

Comment:
string?

Reply:
This is just a dumb ViewModel used to setup Koin. It will be removed once we start to work on home screen